frontend/pages/admin/IntegrationsPage/cards/MdmSettings/VppPage/components/DeleteVppModal/DeleteVppModal.tsx

```diff
@@ -49,7 +49,8 @@ const DeleteVppModal = ({
       <>
         <p>
           Apps purchased for the <b>{orgName}</b> location won&apos;t appear in
-          Fleet. Apps won&apos;t be uninstalled from hosts.
+          Fleet, and app install policy automations will be removed. Apps
```

Comment:
```suggestion
          Fleet, and policies that trigger automatic install of these apps will be removed. Apps
```
I slightly changed this copy to match the language we use on the software title details page.

<img width="365" alt="Screenshot 2025-01-20 at 13 40 36" src="https://github.com/user-attachments/assets/83ec6dec-cf3c-4042-9946-a18fc48d9032" />